src/main/kotlin/com/expedia/graphql/schema/hooks/SchemaGeneratorHooks.kt

```diff
@@ -8,6 +8,9 @@ import kotlin.reflect.KFunction
 import kotlin.reflect.KProperty
 import kotlin.reflect.KType
 
+/**
+ * Collection of all the hooks when generating a schema.
```

Comment:
Maybe define what the hooks are? Something about lifecycle events while the schema is building to allow the user to customize the schema.

Reply:
Updated.